src/dataflow/components/dataflow-program.tsx

```diff
@@ -120,11 +132,15 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
   }
 
   public componentDidUpdate(prevProps: IProps) {
-    if (this.props.size !== prevProps.size) {
-      if (this.programEditor && this.programEditor.view) {
+    if (this.programEditor && this.programEditor.view) {
+      if (this.editorDomElement && this.state.editorContainerWidth !== this.editorDomElement.clientWidth) {
+        this.setState({ editorContainerWidth: this.editorDomElement.clientWidth });
+        this.programEditor.view.resize();
+      } else if (this.props.size !== prevProps.size) {
```

Comment:
I have mixed feelings on this change.  In a previous PR, I added `react-sizeme` so we could monitor parent container size changes and tell the rete editor to update accordingly.  With the side-by-side graph we needed to do the same thing, but internal to this component (rete editor needs to update when user toggles between showing/hiding the graph).  So to handle this, I now store the container width in state and call `this.programEditor.view.resize()` when it changes.  Seems like were now doing too much or there's some overlap between these two ways that we're monitoring size changes.